docs/errors.md

```diff
@@ -48,127 +49,70 @@ Children of `MongoError` include:
 
-_MongoRuntimeError children (pt 1)_
-![(MongoRuntimeError hierarchy tree part 1)](charts/imgs/MongoRuntimeError_0.svg)
-
-_MongoRuntimeError children (pt 2)_
-![(MongoRuntimeError hierarchy tree part 2)](charts/imgs/MongoRuntimeError_1.svg)
-
-_MongoRuntimeError children (pt 3)_
-![(MongoRuntimeError hierarchy tree part 3)](charts/imgs/MongoRuntimeError_2.svg)
-
-Children of `MongoRuntimeError` include:
-
-- #### `MongoTransactionError`
-
-  - Thrown when the user makes a mistake in the usage of transactions (e.g.: attempting to commit a transaction with a readPreference other than primary).
-
-- #### `MongoNotConnectedError`
-
-  - Thrown when the user attempts to operate on the data from a client that has not been connected to a MongoDB server instance.
-
-- #### `MongoKerberosError`
-
-  - Thrown when the user attempts to authenticate via Kerberos, but fails to connect to the Kerberos client.
-
-- #### `MongoCompressionError`
-
-  - Thrown when the driver fails to compress data before sending it to the server.
-
-- #### `MongoDecompressionError`
-
-  - Thrown when the driver fails to decompress data received from the server
-
-- #### `MongoExpiredSessionError`
-
-  - Thrown when the user attempts to operate on a session that has expired or has been closed.
-
-- #### `MongoURIError`
-
-  - Thrown when a user supplies an incorrect URI to the driver.
-
-- #### `MongoResourceClosedError`
+**Children of MongoRuntimeError**
 
-  - Thrown when there is an attempt to access a resource which has already been or will be closed/destroyed.
-    - Children of this error class include:
-      - **`MongoServerClosedError`**: Thrown when an attempt is made to operate on a closed server.
-      - **`MongoStreamClosedError`**: Thrown when an attempt is made to operate on a closed stream.
-      - **`MongoTopologyClosedError`**: Thrown when an attempt is made to operate on a dropped, or otherwise unavailable, database.
+| Error Name                    | Description                                                                               |
+| ----------------------------- | ----------------------------------------------------------------------------------------- |
+| **MongoCompressionError**     | Thrown when the driver fails to compress data before sending it to the server             |
+| **MongoDecompressionError**   | Thrown when the driver fails to decompress data received from the server                  |
+| **MongoChangeStreamError**    | Thrown when an error is encountered when operating on a ChangeStream                      |
+| **MongoGridFSStreamError**    | Thrown when an unexpected state is reached when operating on a GridFS Stream              |
+| **MongoGridFSChunkError**     | Thrown when a malformed or invalid chunk is encountered when reading from a GridFS Stream |
+| **MongoServerSelectionError** | Thrown when the driver fails to select a server to complete an operation                  |
```

Comment:
MongoServerSelectionError extends MongoSystemError and we should probably leave things that way. This should be moved down as a child of MongoSystemError in the md.